components/tools/Checkbox.tsx

```diff
@@ -0,0 +1,66 @@
+import { twMerge } from 'tailwind-merge';
+
+export interface CheckboxProps {
```

Comment:
Add this interface inside types folder and export it from there.

Reply:
Done